less/pages/clubs-form.less

```diff
@@ -4,11 +4,20 @@
     width: 80%;
     margin: auto;
 
+    * {
+      // serious, get out of here, outline.
+      outline: 0 !important;
+    }
```

Comment:
yeah, it's explicitly for the form only. I'm not a fan of !important, except for three things: `.hidden` class, `outline:0` and `user-select:none` =)